src/graphqlHelpers.ts

```diff
@@ -731,7 +742,7 @@ const printObject = (obj: OutObject): string => {
 `
         : "";
 
-      return `${description}${fieldSelection.name}: ${value};`;
+      return `${description}${fieldSelection.name}${fieldSelection.isNullable ? "?" : ""}: ${value};`;
```

Comment:
and use `isNullable` on the field to decide whether to emit `?` or not in the object definition.